package.json

```diff
@@ -15,7 +15,8 @@
     "test:watch": "watch 'npm run test' ./src",
     "test:integration:dist": "node test/integration/dist.test.js",
     "test:integration:npm": "node test/integration/npm.test.js",
-    "build": "npm run test  && rm -rf dist && rollup -c",
+    "rollup": "rm -rf dist && rollup -c",
```

Comment:
As already mentioned I'm hesitant to do any breaking changes but I will be more then happy to merge this change done in package.json If you open new PR with just this change (or leave this PR and undo other changes)

Reply:
That's ok, it's not such a big deal 😛